packages/web-client/app/services/merchant-info.ts

```diff
@@ -21,6 +26,22 @@ interface CheckMerchantSlugUniquenessTaskParams {
 
 export default class MerchantInfoService extends Service {
   @service declare hubAuthentication: HubAuthentication;
+  @service declare offChainJson: OffChainJsonService;
+
+  @task *fetchOffChainMerchantInfo(
```

Comment:
Moved this here so I can use it in the business selector card to fetch merchant info - the slug. And then write it in the workflow session.